src/ts/ConvivaAnalytics.ts

```diff
@@ -62,6 +62,10 @@ export class ConvivaAnalytics {
   // Attributes needed to workaround wrong event order in case of a pre-roll ad. (See #onAdBreakStarted for more info)
   private adBreakStartedToFire: AdBreakEvent;
 
+  // Needed to workaround wrong event order in case of a video-playback-quality-change event. (See
+  // #onVideoQualityChanged for more info)
```

Comment:
The `#` notation does not exist in JS/TS. It's always a `.`, or in this case, `{@link onVideoQualityChanged}`.